src/main/java/org/zephyrsoft/sdb2/remote/MQTT.java

```diff
@@ -71,6 +71,8 @@ public MQTT(String serverUri, String clientID, String userName, String password)
 	@Override
 	public void connectionLost(Throwable cause) {
 		onConnectionLostListeners.forEach(ocll -> ocll.onConnectionLost(cause));
+		// TODO: implement auto reconnect
```

Comment:
Maybe this should be done prior to merging?

Reply:
jeah..I added an issue for that.